docs/notebooks/expected_improvement.pct.py

```diff
@@ -217,10 +217,10 @@ def build_model(data):
 # Here we use the `BatchMonteCarloExpectedImprovement` function. Note that this acquisition function is computed using a Monte-Carlo method (so it requires a `sample_size`), but with a reparametrisation trick, which makes it deterministic.
 
 # %%
-from trieste.acquisition.rule import BatchAcquisitionRule
+from trieste.acquisition.rule import EfficientGlobalOptimization
 
 qei = trieste.acquisition.BatchMonteCarloExpectedImprovement(sample_size=1000)
-batch_rule: BatchAcquisitionRule[Box] = BatchAcquisitionRule(
+batch_rule: EfficientGlobalOptimization[Box] = EfficientGlobalOptimization(
```

Comment:
depending on the final API for `EfficientGlobalOptimization`, we may find we don't need this type hint cos the optimizer can specify the search space

Reply:
OPTIONAL